renovate.json

```diff
@@ -31,9 +31,6 @@
   "ignorePaths": [
     "**/docker/**",
     ".devfile.yaml",
-    ".devfile.yml",
-    "devfile.yaml",
-    "devfile.yml",
```

Comment:
Since all of these could be used in either case we should fix them to only exclude any of these at the root level rather than any pattern that matches these.

@thepetk Would the following changes work?
```diff
-    ".devfile.yaml",
-    ".devfile.yml",
-    "devfile.yaml",
-    "devfile.yml",
+    "/.devfile.yaml",
+    "/.devfile.yml",
+    "/devfile.yaml",
+    "/devfile.yml",
```

Reply:
Yeap I think this is a better alternative than creating a separate dir :) @michael-valdron

Reply:
done!